sympy/logic/boolalg.py

```diff
@@ -719,6 +719,12 @@ def _eval_as_set(self):
         from sympy.sets.sets import Intersection
         return Intersection(*[arg.as_set() for arg in self.args])
 
+    def _rewrite_as_Nor(self):
+        expr = 0
+        for arg in self.args:
```

Comment:
It should be possible to write this on a single line.  Like `Nor(*[Not(arg) for arg in self.args])`